src/main/java/calculator/Application.java

```diff
@@ -1,7 +1,53 @@
 package calculator;
 
+import camp.nextstep.edu.missionutils.Console;
+
+import java.util.ArrayList;
+
 public class Application {
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+        //입력 받는 객체 scanner 생성
+        System.out.println("덧셈할 문자열을 입력해주세요.");
+        String input = Console.readLine();
+
+        //구분자 추가하기
+        String op = ",|:";
+        //새로 등록할 구분자 있나 확인
+        if (input.length() >= 5) {
+            String front = input.substring(0, 2);
+            String back = input.substring(3, 5);
+            //제대로된 형식 사이에 구분자가 끼어있는지 verify
+            if (front.equals("//") && back.equals("\\n")) {
+                op += "|";
+                op += Character.toString(input.charAt(2));
+                input = input.substring(5);
+            }
+        }
+
+        // 구분자 제외하고 남은 숫자 배열에 담는 기능
+        String[] number = input.split(op);
+        ArrayList<String> numbers = new ArrayList<>();
+
+        for(int i = 0; i < number.length; i++) {
+            if(!number[i].equals("")){
+                numbers.add(number[i]);
+            }
+        }
+
+        // 배열에 담겨있는 숫자 더하는 기능
+        stringCal cal = new stringCal();
+        String sum = "0";
+        for (String i : numbers) {
+            for(int j = 0; j < i.length(); j++){
+                if(!Character.isDigit(i.charAt(j))){
+                    IllegalArgumentException e = new IllegalArgumentException();
+                    throw e;
+                }
+            }
```

Comment:
자바 코드 컨벤션에 맞게 작성하면 좋겠네요! Cmd Option L 단축키로 코드 정리 한번씩 해주세요
변수 선언은 여기서는 굳이 필요 없어보여요.
아마 인텔리제이에서도 경고 띄워주고 있을텐데 한번씩 읽어보면 좋을 것 같아요

```suggestion
            for (int j = 0; j < i.length(); j++) {
                if (!Character.isDigit(i.charAt(j))) {
                    throw new IllegalArgumentException();
                }
            }
```